src/Nancy/Helpers/ProxyNancyReferenceProber.cs

```diff
@@ -0,0 +1,18 @@
+#if !CORE
+namespace Nancy.Helpers
+{
+    using System;
+    using System.Reflection;
+    using Nancy.Extensions;
+
+    internal class ProxyNancyReferenceProber : MarshalByRefObject

```

Comment:
@thecodejunkie I don't see which access modifier you're referring to? Besides making it `public`.